app/components/Home.vue

```diff
@@ -62,7 +63,7 @@ export default {
   },
   methods: {
     changeTabTo (event) {
-      this.$store.commit('SET_CURRENT_TAB', event.newIndex)
+      this.$store.commit(ActionTypes.SET_CURRENT_TAB, event.newIndex)
     },
     bottomNavigationLoaded (argv) {
       // cuando se recibe un msj con la aplicacion en background
```

Comment:
I think this comment is not describing what bottomNavigationLoaded does, but something else. I'd remove it.

Reply:
I agree